broker/cmd/dejaqcli-broker/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/ilyakaznacheev/cleanenv"
+
+	"github.com/dejaq/prototype/broker/pkg/carrier"
+	"github.com/dejaq/prototype/broker/pkg/overseer"
+	"github.com/dejaq/prototype/broker/pkg/storage/cockroach"
+	"github.com/dejaq/prototype/broker/pkg/storage/inmemory"
+	"github.com/dejaq/prototype/broker/pkg/storage/redis"
+	storageTimeline "github.com/dejaq/prototype/broker/pkg/storage/timeline"
+	"github.com/dejaq/prototype/grpc/DejaQ"
+	flatbuffers "github.com/google/flatbuffers/go"
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+type Config struct {
+	Host         string `env:"HOST" env-default:"127.0.0.1:9000"`
```

Comment:
This is usually called binding address, the host is the actually machine that is running on